uint8[] enable
uint8[] opMode
float64[] kp
float64[] kd
float64[] q
float64[] qd
float64[] cur

